src/style.css

```diff
@@ -0,0 +1,81 @@
+* {
+  box-sizing: border-box;
+}
+
+@font-face {
+  font-family: "Avenir";
+  src: url(./fonts/Avenir-Book.ttf);
+  font-weight: 300;
+}
+
+.search-input_box:first-child {
+  max-width: 100%;
+  margin: 20px 0;
+  position: relative;
+}
+
+.search-input_box:last-child {
+  max-width: 100%;
+  margin: 20px 0;
+  position: relative;
+}
```

Comment:
This class is always first and last child (it's only child). Just use class, no need to overcomplicate